src/CMakeLists.txt

```diff
@@ -224,6 +225,7 @@ if( ICONV_FOUND )
     target_link_libraries( exiv2lib PRIVATE Iconv::Iconv )
 endif()
 
+write_basic_package_version_file(exiv2ConfigVersion.cmake COMPATIBILITY ExactVersion)
```

Comment:
The question to be answered: is compatibility mode set to 'ExactVersion' ok? Do we have any rules about version compatibility?

Reply:
For me it's fine like this, since users compiling and deploying this version of the library will want to use this specific version. It also works if the consumer does not provide the version information:

```
find_package(exiv2lib REQUIRED CONFIG NAMES exiv2)
```

Reply:
Great :)

Reply:
That's what I call efficiency! :+1: